### PR TITLE
test: add migrationVerify test

### PR DESCRIPTION
**Pull request by nguyen102**
Issue #, if available:

Description of changes:
Added unit tests for `migrationVerify.ts`
```
--------------------|---------|----------|---------|---------|-------------------------------------------------
File                | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s                               
--------------------|---------|----------|---------|---------|-------------------------------------------------
All files           |   76.59 |    54.73 |   78.57 |   76.64 |                                                 
 binaryConverter.ts |    73.8 |    55.55 |    62.5 |   74.69 | 45,66,77,108-109,129-132,166-187                
 exportHelper.ts    |       0 |        0 |       0 |       0 | 8-131                                           
 migrationExport.ts |     100 |      100 |     100 |     100 |                                                 
 migrationImport.ts |   80.27 |       50 |   81.81 |   79.72 | 63-66,78,85,129-141,145,165,261,276-277,312-339 
 migrationUtils.ts  |   85.41 |    68.42 |      90 |   86.51 | 36,92-113,152-166                               
 migrationVerify.ts |    85.5 |       50 |   83.33 |   86.15 | 72,102,114-122,145-146                          
--------------------|---------|----------|---------|---------|-------------------------------------------------

Tests finished:
  Successes: 34
  Failures: 0
  Total: 34
```

Checklist:

- [ ] Have you successfully deployed to an AWS account with your changes?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Did you add new integration tests?
- [ ] Did you run integration tests with your changes?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: fwoa-tools/src/migrationVerify.test.ts [hunk 1]
```diff
@@ -0,0 +1,144 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+jest.mock('./migrationUtils', () => ({
+    checkConfiguration: () => {},
+    getFhirClientSMART: () => {
+        //eslint-disable-next-line
```
Comment (Bingjiling) [inline fwoa-tools/src/migrationVerify.test.ts:9]: Nit: specify which rule is disabled here

File: fwoa-tools/src/migrationVerify.test.ts [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        return axios.create();
+    },
+    getFhirClient: () => {
+        //eslint-disable-next-line
+        return axios.create();
+    }
+}));
+
+import AWS from 'aws-sdk';
+import {GetObjectRequest} from "aws-sdk/clients/s3";
```
Comment (Bingjiling) [inline fwoa-tools/src/migrationVerify.test.ts:19]: Nit: the formatting here looks off, should it be `import { GetObjectRequest } from "aws-sdk/clients/s3"`?

File: fwoa-tools/src/migrationVerify.test.ts [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+                '-d',
+            ];
+
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            const argv: any = parseCmdOptions();
+            expect(argv.dryRun).toEqual(true);
+            expect(argv.smart).toEqual(true);
+        });
+        test('smart and dryrun not enabled', () => {
+            process.argv = [
```
Comment (Bingjiling) [inline fwoa-tools/src/migrationVerify.test.ts:62]: Do we need to restore `process.argv` in the end?

File: fwoa-tools/src/migrationVerify.test.ts [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+            'migrationExport.ts'
+        ];
+        const {smartClient, dryRun} = buildRunScriptParams();
+        expect(dryRun).toEqual(false);
+        expect(smartClient).toEqual(false);
+    })
+
+    test('runScript', async() => {
+        process.env.EXPORT_BUCKET_NAME = 'fake-bucket-name'
+        const fakeFileBody =
```
Comment (Bingjiling) [inline fwoa-tools/src/migrationVerify.test.ts:85]: Nit: it'll be easier to read/update if we use `JSON.stringify(<the actual JSON object>)` here.

**Review by Bingjiling — APPROVED** [review]

File: fwoa-tools/src/migrationVerify.test.ts (hunk 3 above)
Comment (nguyen102) [reply fwoa-tools/src/migrationVerify.test.ts:62]: I'll update the `afterEach` to restore `process.argv`

File: fwoa-tools/src/migrationVerify.test.ts (hunk 1 above)
Comment (nguyen102) [reply fwoa-tools/src/migrationVerify.test.ts:9]: updated

File: fwoa-tools/src/migrationVerify.test.ts (hunk 4 above)
Comment (nguyen102) [reply fwoa-tools/src/migrationVerify.test.ts:85]: updated

**Pull request merged**